### PR TITLE
NTella food, drinks, and crepes

### PR DESCRIPTION
**Pull request by tomixcomics**
Added some foods and drinks using the new choco-nut spread ingredient.
Made several different kinds of crepes.

### CODE REVIEW

**Comment by tomixcomics** [comment]
!review required

**Review by FluffyGhoster — APPROVED** [review]

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 1]
```diff
@@ -2407,6 +2407,28 @@
 	glass_name = "mug of apple cider juice"
 	glass_desc = "It's just spiced up apple juice. Sometimes the barista can't work miracles."
 
+/singleton/reagent/drink/notellamilkshake
+	name = "Notella Milkshake"
```
Comment (NonQueueingMatt) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:2411]: not a fan of the name "notella", find an appropriate replacement

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 2]
```diff
@@ -2407,6 +2407,28 @@
...
+	color = "#6d4124"
+	taste_description = "overwhelmingly sweet chocolate"
+
+	glass_icon_state = "notellamilkshake"
+	glass_name = "glass of notella milkshake"
+	glass_desc = "Oh look, it's that thing you actually want to get but probably shouldn't."
+	glass_center_of_mass = list("x"=16, "y"=7)
+
+/singleton/reagent/drink/notellahotchocolate
+	name = "Notella Hot Chocolate"
```
Comment (NonQueueingMatt) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:2422]: not a fan of the name "notella", find an appropriate replacement

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 3]
```diff
@@ -2407,6 +2407,28 @@
...
+	description = "It's like a cup of hot chocolate except... More everything."
+	color = "#63432e"
+	taste_description = "hazelnutty, creamy chocolate"
+
+	glass_icon_state = "notellahotchocolate"
+	glass_name = "glass of notella hot chocolate"
+	glass_desc = "A very chocolatey drink for the days so rough, so cold, or so celebratory that a regular hot hcocolate just won't cut it. It has marshmallows!"
```
Comment (NonQueueingMatt) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:2429]:
```suggestion
	glass_desc = "A very chocolatey drink for the days so rough, so cold, or so celebratory that a regular hot chocolate just won't cut it. It has marshmallows!"
```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 4]
```diff
@@ -2407,6 +2407,28 @@
...
 	glass_desc = "It's just spiced up apple juice. Sometimes the barista can't work miracles."
 
+/singleton/reagent/drink/notellamilkshake
+	name = "Notella Milkshake"
+	description = "An intensely sweet chocolatey concoction with whipped cream on top."
+	color = "#6d4124"
+	taste_description = "overwhelmingly sweet chocolate"
+
+	glass_icon_state = "notellamilkshake"
+	glass_name = "glass of notella milkshake"
```
Comment (NonQueueingMatt) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:2417]: not a fan of the name "notella", find an appropriate replacement

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 5]
```diff
@@ -3012,6 +3012,20 @@
...
 
+/datum/chemical_reaction/drink/notellashake
+	name = "notella milkshake"
+	id = "notellamilkshake"
+	result = /singleton/reagent/drink/notellamilkshake
+	required_reagents = list(/singleton/reagent/drink/milkshake = 5, /singleton/reagent/nutriment/choconutspread = 1)
+	result_amount = 6
+
+/datum/chemical_reaction/drink/notellahotchocolate
+	name = "notella hot chocolate"
```
Comment (NonQueueingMatt) [inline code/modules/reagents/Chemistry-Recipes.dm:3023]: not a fan of the name "notella", find an appropriate replacement

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 6]
```diff
@@ -3012,6 +3012,20 @@
 	catalysts = list(/singleton/reagent/enzyme = 5)
 	result_amount = 1
 
+/datum/chemical_reaction/drink/notellashake
+	name = "notella milkshake"
```
Comment (NonQueueingMatt) [inline code/modules/reagents/Chemistry-Recipes.dm:3016]: not a fan of the name "notella", find an appropriate replacement

File: code/modules/reagents/reagent_containers/food/snacks/confections.dm [hunk 7]
```diff
@@ -135,3 +135,66 @@
...
+//crepe dishes
+
+/obj/item/reagent_containers/food/snacks/crepe
+	name = "crepe"
+	desc = "A very thin pancake."
+	trash = /obj/item/trash/plate
+	icon = 'icons/obj/item/reagent_containers/food/confections.dmi'
+	filling_color = "#d4b28b"
+	bitesize = 2
+/obj/item/reagent_containers/food/snacks/crepe/chocolate
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/food/snacks/confections.dm:148]:
```suggestion
	bitesize = 2

/obj/item/reagent_containers/food/snacks/crepe/chocolate
```

File: code/modules/reagents/reagent_containers/food/snacks/pastries.dm [hunk 8]
```diff
@@ -415,6 +415,28 @@
 	reagents_to_add = list(/singleton/reagent/nutriment = 3)
 	reagent_data = list(/singleton/reagent/nutriment = list("cake" = 5, "sweetness" = 5, "apple" = 5))
 
+/obj/item/reagent_containers/food/snacks/sliceable/cake/notellacheesecake
+	name = "notella cheesecake"
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/food/snacks/pastries.dm:419]: not a fan of the name "notella", find an appropriate replacement

File: code/modules/reagents/reagent_containers/food/snacks/pastries.dm [hunk 9]
```diff
@@ -415,6 +415,28 @@
...
+	desc = "An elaborate layer cheesecake made with chocolate hazelnut spread. You gain calories just by looking at it for too long."
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/food/snacks/pastries.dm:420]:
```suggestion
	desc = "An elaborate layered cheesecake made with chocolate hazelnut spread. You gain calories just by looking at it for too long."
```

File: code/modules/reagents/reagent_containers/food/snacks/pastries.dm [hunk 10]
```diff
@@ -415,6 +415,28 @@
...
+	icon_state = "notellacheesecake"
+	slice_path = /obj/item/reagent_containers/food/snacks/cakeslice/notellacheesecake_slice
+	slices_num = 5
+	filling_color = "#331c03"
+	center_of_mass = list("x"=16, "y"=10)
+	reagents_to_add = list(/singleton/reagent/nutriment = 20)
+	reagent_data = list(/singleton/reagent/nutriment = list("hazelnut chocolate" = 15, "creamy cheese" = 10, "crunchy cookie base" = 5))
+
+/obj/item/reagent_containers/food/snacks/cakeslice/notellacheesecake_slice
+	name = "notella cheesecake slice"
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/food/snacks/pastries.dm:431]: not a fan of the name "notella", find an appropriate replacement

File: code/modules/reagents/reagent_containers/food/snacks/pastries.dm [hunk 11]
```diff
@@ -415,6 +415,28 @@
...
+	desc = "A slice of cake marrying the chocolate taste of notella with the creamy smoothness of cheesecake, all on a cookie crumble base."
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/food/snacks/pastries.dm:432]: not a fan of the name "notella", find an appropriate replacement

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm (hunk 1 above)
Comment (tomixcomics) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:2411]: Not sure what the issue is... "Notella" is already the name of the spread in game (the reagent is called choco-nut spread but the bottle is refferred to as a notella jar, there's already a food item in the game called notella bread slice, and i've also gotten several positive comments about it from people who saw it in the game. )

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**